### ISSUE TITLE
auto_train JSON error

### ISSUE THREAD

**Issue by skunkwerk** [ludwig-ai/ludwig#3198]
**Describe the bug**
Running auto_train with a sample dataset generates the error: ludwig.error.ConfigValidationError: Failed to validate JSON schema for config. Error: Additional properties are not allowed ('decay_rate', 'decay_steps' were unexpected)

**To Reproduce**

```
from ludwig.automl import auto_train
from ludwig.datasets import mushroom_edibility
from ludwig.utils.dataset_utils import get_repeatable_train_val_test_split

mushroom_df = mushroom_edibility.load()
mushroom_edibility_df = get_repeatable_train_val_test_split(mushroom_df, 'class', random_seed=42)

auto_train_results = auto_train(
    dataset=mushroom_edibility_df,
    target='class',
    time_limit_s=250,
    tune_for_memory=False,
    user_config={'preprocessing': {'split': {'column': 'split', 'type': 'fixed'}}},
)
```

**Expected behavior**
No errors.

**Environment (please complete the following information):**

- OS:  Linux
- Version: 4.15.0-189-generic
- Python version: 3.8
- Ludwig version: 0.7.1

thanks

**Comment by tgaddair** [comment]
Thanks for reporting @skunkwerk, I was able to repro the issue locally. We'll try and get a fix in for v0.7.2.

**Comment by tgaddair** [comment]
Verified a fix in #3203.

**Comment by tgaddair** [comment]
Hey @skunkwerk, we just released v0.7.2 which should contain a fix for this issue. Let me know if you run into any additional issues!

**Comment by skunkwerk** [comment]
awesome, thanks Travis.
